build.gradle

```diff
@@ -79,8 +79,8 @@ subprojects {
 
     compileClasspath {
       // do not exclude Guava so the bundle project can reference classes.
-      // the Spark module is also excluded because this breaks the Scala compiler
-      if (project.name != 'iceberg-bundled-guava' && project.name != 'iceberg-spark' && project.name != 'iceberg-spark2') {
+      // the Spark2 module is also excluded because this breaks the Scala compiler
+      if (project.name != 'iceberg-bundled-guava' && project.name != 'iceberg-spark2') {
```

Comment:
The spark2 module was included so that the relocated Scala code would work. If we no longer have any Scala, then we don't need this for spark2 either.

Reply:
Thanks. I'll remove that.

Reply:
This is done.